mibs/carel/CAREL-RITTAL-LCP-3311-MIB.mib

```diff
@@ -0,0 +1,767 @@
+-- *****************************************************************

```

Comment:
Rename this file to `CAREL-RITTAL-LCP-3311-MIB` without .mib please

Reply:
done :)